sktime/datatypes/_series/_convert.py

```diff
@@ -142,9 +142,12 @@ def convert_np_to_MvS_as_Series(obj: np.ndarray, store=None) -> pd.DataFrame:
 def convert_np_to_UvS_as_Series(obj: np.ndarray, store=None) -> pd.Series:
 
     if not isinstance(obj, np.ndarray) and len(obj.shape) < 3:
```

Comment:
```suggestion
    if not isinstance(obj, np.ndarray) and obj.ndim < 3:
```

Reply:
Why do we test for `isinstance(obj, np.ndarray)` here but not in the other if-condition below? Perhaps we should have an `isinstance` check at the beginning since that's what the docstring specifies? I'm not sure if this is meant to be a public function, but passing objects without a shape attribute will result in an AttributeError (e.g. passing a list).

I would find this more readable:

```python
if not isinstance(obj, np.ndarray):
    raise ValueError(...)

if obj.ndim > 2:
   raise ValueError(...)

if obj.ndim == 2 and obj.shape[1] > 1:
    raise ValueError(...)
```

Reply:
This is a typo, the line should say:
`if not isinstance(obj, np.ndarray) or len(obj.shape) > 2:`

Reply:
if it's `or`, the second statement will only be evaluated if the first statement is false, so it's fine

Reply:
I'd prefer the three conditions above, more readable